components/eamxx/src/physics/p3/tests/CMakeLists.txt

```diff
@@ -74,7 +74,9 @@ if (SCREAM_ENABLE_BASELINE_TESTS)
     ${FORCE_RUN_DIFF_FAILS})
 endif()
 
-if (NOT SCREAM_P3_SMALL_KERNELS)
+# If small kernels are ON, we don't need a separate executable to test them.
+# Also, we never want to generate baselines with this separate executable
+if (NOT SCREAM_P3_SMALL_KERNELS AND NOT SCREAM_ONLY_GENERATE_BASELINES)
```

Comment:
I'm a bit puzzled by our sk tests: they use the same tests source files...and I don't see any mention of SK in the cpp files in the tests folder. Are we just running non-SK tests twice? I think we are. Where is `p3_sk_tests` triggering the call of p3_sk stuff instead of "regular" p3 stuff?

Reply:
I added `  printf("cloud_sed_disp!\n");` in `p3_cloud_sed_impl_disp.cpp`, and ran `p3_sk_tests`. It never printed.